scripts/dist2wheel.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
```

Comment:
Could you rename `scripts` to `python` instead?  For a go project I kinda want to make it clear this directory is specifically for our python packaging vs a more generic scripts dir.

Reply:
Done.